src/Robo/Plugin/Commands/VrtRunCommand.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Fire\Robo\Plugin\Commands;
+
+use Robo\Symfony\ConsoleIO;
+use Robo\Robo;
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * Provides a command to run the VRT testing.
+ */
+class VrtRunCommand extends FireCommandBase {
+
+  /**
+   * Runs your VRT testing (lando Only)
+   *
+   * Usage Example: fire vrt-run
+   *
+   * @command vrt:run
+   * @aliases vrun
+   *
+   */
+  public function vrtRun(ConsoleIO $io) {
+    $env = Robo::config()->get('local_environment');
+    $tasks = $this->collectionBuilder($io);
+    if ($env == 'lando') {
+      $reconfigureTestingUrls = $io->choice('Do you want yo reconfigure your reference and test urls?', ['Yes', 'No']);
+      $newReferenceFiles = $io->choice('Do you want to re-take the reference screenshots?', ['Yes', 'No']);
+      if (strtolower($reconfigureTestingUrls) === 'yes') {
+        $tasks->addTask($this->taskExec('fire vrt:testing-setup'));
+      }
+      if (strtolower($newReferenceFiles) === 'yes') {
+        $tasks->addTask($this->taskExec('fire vrt:reference'));
+      }
+      $landoConfig = Yaml::parse(file_get_contents($this->getLocalEnvRoot() . '/.lando.yml'));
+
+      $tasks->addTask($this->taskExec($env . ' ssh -s backstopserver -c "cd /app/tests/backstop && backstop test --config=/app/tests/backstop/backstop-local.json"'));
+      $tasks->addTask($this->taskOpenBrowser('https://' . $landoConfig['name'] . '.lndo.site/backstop/html_report/index.html'));
```

Comment:
This appears to be the only part that didn't work smoothly for me. The first command returned an error which I think can be expected if a test fails:

![CleanShot 2024-06-13 at 12 19 04@2x](https://github.com/fourkitchens/fire/assets/8762630/e2e72fca-5b96-4e5e-aa05-b6033231ad47)


That error though prevented the taskOpenBrowser task from running it seems

Reply:
Thank you , That  also makes sense, For this command I'm going to not use the task builder